viewing_party/party.py

```diff
@@ -1,23 +1,197 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movies = {}
+    if title != None and genre != None and rating != None:
+        movies= {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return movies
+    else:
+        return None
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data

```

Comment:
Something to consider is, what happens if a user tries to add a movie to these lists that is already present? Maybe we could implement some type of guard clause?